### ISSUE TITLE
How to include archived items?

### ISSUE THREAD

**Issue by Yolakalemowa** [mmagnus/Pocket-Plus-Calibre-Plugin#51]
Hi,

I hope I'm not mistaken or missed in the documentation but I didn't find a way to sync the archived items...

is there a way to include them?

thanks a lot for a great plugin

**Comment by mmagnus** [comment]
Dear @Yolakalemowa, you can change TO_PULL to 'all', (so in python `TO_PULL='all'`), and then all articles will be considered to be converted into a book (but not specifically only archived, if this is what you want, getting only archived is impossible at the moment I guess.)

       **TO_PULL** ('all' or 'unread') What articles to pull? unread only or all?

**Comment by Yolakalemowa** [comment]
thank you @mmagnus sorry I missed that obvious one. I didn't know items in main list are called "unread".

**Comment by mmagnus** [comment]
Yeah, I agree this is not the most obvious naming. I hope this will work, let me know if not. Enjoy!
